### PR TITLE
Case 22196: HMD JSDoc update

### PR DESCRIPTION
**Pull request by ctrlaltdavid**
Namepaces:
- HMD

Manuscript case: https://highfidelity.manuscript.com/f/cases/22196/JSDoc-HMD

Note: No executable code changes.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18711/HighFidelity-Beta-PR15383-1906ce7.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18711/HighFidelity-Quest-Beta-PR15383-1906ce7.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18711/HighFidelity-Beta-PR15383-1906ce7.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18711/HighFidelity-Beta-Interface-PR15383-1906ce7.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18711/HighFidelity-Beta-PR15383-1906ce7.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18711/HighFidelity-Beta-Interface-PR15383-1906ce7.exe

File: interface/src/scripting/HMDScriptingInterface.h [hunk 1]
```diff
@@ -292,7 +292,7 @@ class HMDScriptingInterface : public AbstractHMDScriptingInterface, public Depen
 
 
     /**jsdoc
-     * Suppress the activation of the HMD-provided keyboard, if any. Successful calls should be balanced with a call to 
+     * Suppresses the activation of the HMD-provided keyboard, if any. Successful calls should be balanced with a call to 
      * {@link HMD.unspressKeyboard|unspressKeyboard} within a reasonable amount of time.
```
Comment (nimisha20) [inline interface/src/scripting/HMDScriptingInterface.h]: This link doesn't seem to be working for me.

File: interface/src/scripting/HMDScriptingInterface.h [hunk 2]
```diff
@@ -172,29 +172,29 @@ class HMDScriptingInterface : public AbstractHMDScriptingInterface, public Depen
...
      */
     Q_INVOKABLE QString preferredAudioOutput() const;
 
 
     /**jsdoc
-     * Check whether there is an HMD available.
+     * Checks whether there is an HMD available.
      * @function HMD.isHMDAvailable
      * @param {string} [name=""] - The name of the HMD to check for, e.g., <code>"Oculus Rift"</code>. The name is the same as 
      *     may be displayed in Interface's "Display" menu. If no name is specified then any HMD matches.
```
Comment (nimisha20) [inline interface/src/scripting/HMDScriptingInterface.h]:
```suggestion
     *     may be displayed in Interface's "Display" menu. If no name is specified, then any HMD matches.
```

File: interface/src/scripting/HMDScriptingInterface.h [hunk 3]
```diff
@@ -208,7 +208,7 @@ class HMDScriptingInterface : public AbstractHMDScriptingInterface, public Depen
     Q_INVOKABLE bool isHMDAvailable(const QString& name = "");
 
     /**jsdoc
-     * Check whether there is an HMD head controller available.
+     * Checks whether there is an HMD head controller available.
      * @function HMD.isHeadControllerAvailable
      * @param {string} [name=""] - The name of the HMD head controller to check for, e.g., <code>"Oculus"</code>. If no name is 
      *     specified then any HMD head controller matches.
```
Comment (nimisha20) [inline interface/src/scripting/HMDScriptingInterface.h]:
```suggestion
     *     specified, then any HMD head controller matches.
```

File: interface/src/scripting/HMDScriptingInterface.h (hunk 1 above)
Comment (ctrlaltdavid) [reply interface/src/scripting/HMDScriptingInterface.h]: Fixed.

File: interface/src/scripting/HMDScriptingInterface.h (hunk 2 above)
Comment (ctrlaltdavid) [reply interface/src/scripting/HMDScriptingInterface.h]: Done.

File: interface/src/scripting/HMDScriptingInterface.h (hunk 3 above)
Comment (ctrlaltdavid) [reply interface/src/scripting/HMDScriptingInterface.h]: Done, plus one further occurrence.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19040/HighFidelity-Beta-PR15383-eff97d4.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19040/HighFidelity-Beta-Interface-PR15383-eff97d4.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19040/HighFidelity-Beta-PR15383-eff97d4.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19040/HighFidelity-Quest-Beta-PR15383-eff97d4.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19040/HighFidelity-Beta-PR15383-eff97d4.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19040/HighFidelity-Beta-Interface-PR15383-eff97d4.exe

**Pull request merged**